src/inc/crsttypes.h

```diff
@@ -1,6 +1,8 @@
+//
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
+//
```

Comment:
The CrstTypeTool added it. There wasn’t a license header before and I think that is a good thing.